lib/Search/SearchResults.php

```diff
@@ -212,7 +212,13 @@ protected function getRelatedId(SugarBean $obj, string $idName, string $link): s
             $relId = $obj->$link->getFocus()->$relField;
             if (is_object($relId)) {
                 if (method_exists($relId, "getFocus")) {
-                    $relId = $relId->getFocus()->id;
```

Comment:
Hi @SinergiaCRM, thank for your PR.

Question: is `getFocus()` ever a loaded bean? Just thinking that maybe if there are scenarios where it is set, we could avoid loading the relationship?

Reply:
Hi @clemente-raposo ,

Sorry, I'm not sure how to check that. Do you have a suggestion?